src/Fabulous/Cmd.fs

```diff
@@ -170,6 +214,8 @@ module Cmd =
         let inline attempt (task: 'a -> Async<_>) (arg: 'a) (ofError: _ -> 'msg) : Cmd<'msg> =
             OfAsyncWith.attempt Async.StartImmediate task arg ofError
 
+    //TODO what do I use this for? Compared to other OfAsync... modules?
```

Comment:
Most .NET libraries use `Task` rather than `Async` which is F#-only. `OfTask` is here when it's easier to just use `task` compared to `async`. Also, tasks are "hot" and will execute immediately as soon as they are defined compared to async which will wait to be executed (with `let!` or `Async.Start`, etc.).

Reply:
Thanks for clarifying and especially for pointing out the difference in "hotness" - I understand F# better now :)
I'll update the docs with this info.